Lib/test/test_ctypes/test_aligned_structures.py

```diff
@@ -58,45 +49,52 @@ class SomeBools(base):
                 ]
             class Main(base):
                 _fields_ = [
-                    ("x", SomeBools),
-                    ("y", c_uint32),
+                    ("x", c_ubyte),
+                    ("y", SomeBools),
+                    ("z", c_uint32),
```

Comment:
What if make `z` a byte and make `SomeBools` containing only 2 fields? Then without the `_align_` setting `Main` can be packed in just 4 bytes. The `_align_` setting will made larger difference.

Or just set `_align_` to 8. This would cover also the next test.